### PR TITLE
Message Orchestration/Canvas Sync Alt Text

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the chart below to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then `Docs`, this can take up to 5 minutes to load. A `502` error means the deployment is still loading, refresh the page, and try again.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Application Infrastructure<br>Data Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Intelligence<br>Product Partnerships<br>SMS<br>Internal Tools</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Ingestion<br>Platforms and Channels<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Messaging and Automation<br>Dashboard Infrastructure<br>Email</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ Select the desired Facebook ad account. Under the __Choose a New or Existing Aud
 __Create a New Audience__<br>
 Enter a name for the new custom audience, select __Add Users to Audience__ and select which fields you would like to sync with Facebook. Next, save your audience by clicking the __Create Audience__ button at the bottom of the step editor.
 
-![Facebook Sync]({% image_buster /assets/img/fb_audience_sync/create_audience.png %})
+![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account is selected and a new audience is created.]({% image_buster /assets/img/fb_audience_sync/create_audience.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:84]:
```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account is selected and a new audience is created.]({% image_buster /assets/img/fb_audience_sync/create_audience.png %})
```

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 2]
```diff
@@ -81,11 +81,11 @@ Select the desired Facebook ad account. Under the __Choose a New or Existing Aud
...
 
 Users will be notified at the top of the step editor if the audience is created successfully or if errors arise during this process. Users can also reference this audience for user removal later in the Canvas journey because the audience was created in draft mode. 
 
-![Facebook Sync]({% image_buster /assets/img/fb_audience_sync/new_audience.png %})
+![An image of the alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/fb_audience_sync/new_audience.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:88]:
```suggestion
![An alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/fb_audience_sync/new_audience.png %})
```

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ When you launch a Canvas with a new audience, Braze will create the new custom a
 __Sync with an Existing Audience__<br>
 Braze also offers the ability to either add or remove users from existing Facebook custom audiences to ensure that these audiences are up-to-date. To sync with an existing audience, type the existing audience name in the dropdown and choose whether you want to __Add to the Audience__ or __Remove from the Audience__. Braze will then either add or remove users in near real-time as they enter the Facebook Audience Step. 
 
-![Facebook Sync]({% image_buster /assets/img/fb_audience_sync/add_audience.png %})
+![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account and existing audience is selected.]({% image_buster /assets/img/fb_audience_sync/add_audience.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:97]:
```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account and existing audience are selected.]({% image_buster /assets/img/fb_audience_sync/add_audience.png %})
```

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ Once you have configured your Facebook Audience Step, simply launch the Canvas!
 
 The __History__ tab of the custom audience in the Facebook Audience Manager will reflect the number of users sent to the audience from Braze. If a user re-enters the step, they will be sent to Facebook again.
 
-![Audience History][9]{: style="max-width:80%;"}
+![An image of what the audience deatils and history looks like for a given Facebook audience.][9]{: style="max-width:80%;"}
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:110]:
```suggestion
![Audience details and the History tab for a given Facebook audience that includes an Audience History table with columns for the activity, activity details, items changed, and the date and time.][9]{: style="max-width:80%;"}
```

File: _docs/_partners/canvas_steps/google_audience_sync.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ Enter a name for the new custom audience, select __Add Users to Audience__ and s
 
 Next, save your audience by clicking the __Create Audience__ button at the bottom of the step editor.
 
-![Google Sync]({% image_buster /assets/img/google_sync/google_sync7.png %})
+![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account is selected, a new audience is created, and the "customer contact info" check box selected.]({% image_buster /assets/img/google_sync/google_sync7.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/google_audience_sync.md:79]:
```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account is selected, a new audience is created, and the "customer contact info" checkbox is selected.]({% image_buster /assets/img/google_sync/google_sync7.png %})
```

File: _docs/_partners/canvas_steps/google_audience_sync.md [hunk 6]
```diff
@@ -76,11 +76,11 @@ Enter a name for the new custom audience, select __Add Users to Audience__ and s
...
 
 Users will be notified at the top of the step editor if the audience is created successfully or if errors arise during this process. Users can reference this audience for user removal later in the Canvas journey because the audience was created in draft mode. 
 
-![Google Sync]({% image_buster /assets/img/google_sync/google_sync9.png %})
+![An image of the alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/google_sync/google_sync9.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/google_audience_sync.md:83]:
```suggestion
![Alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/google_sync/google_sync9.png %})
```

File: _docs/_partners/canvas_steps/google_audience_sync.md [hunk 7]
```diff
@@ -94,7 +94,7 @@ Braze also offers the ability to either add or remove users from existing Google
 
 Once you've configured your Google Audience step, select __Done__. Your Google Audience step will include details about the new audience.
 
-![Google Sync]({% image_buster /assets/img/google_sync/google_sync8.png %})
+![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account and existing audience is selected, as well as the "Add user to Audience" radio button.]({% image_buster /assets/img/google_sync/google_sync8.png %})
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/google_audience_sync.md:97]:
```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account and existing audience is selected, as well as the "Add user to Audience" radio button.]({% image_buster /assets/img/google_sync/google_sync8.png %})
```

File: _docs/_partners/message_orchestration/additional_channels/direct_mail/inkit.md [hunk 8]
```diff
@@ -35,13 +35,13 @@ Check out [Inkit documentation](https://help.inkit.com/hc/en-us/articles/3600365
 
 To create an Inkit webhook template to use in future campaigns or Canvases, navigate to the **Templates & Media** section in the Braze platform. If you would like to create a one-off Inkit webhook campaign or use an existing template, select **Webhook** in Braze when creating a new campaign.
 
-![Inkit webhook template][7]
+![An image of some of the available predesigned webhook templates in Braze.][7]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/direct_mail/inkit.md:38]:
```suggestion
![A selection of available predesigned webhook templates in the Webhook Templates tab of the Templates & Media section.][7]
```

File: _docs/_partners/message_orchestration/additional_channels/direct_mail/inkit.md [hunk 9]
```diff
@@ -35,13 +35,13 @@ Check out [Inkit documentation](https://help.inkit.com/hc/en-us/articles/3600365
...
 
 Once you have selected the Inkit webhook template, you should see the following:
 - **Webhook URL**: `https://internal.inkit.io/integrations/webhook`
 - **Request Body**: Raw Text
 
-![Inkit integration][5]
+![An image of the request body code and webhook URL shown in the Braze webhook builder compose tab.][5]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/direct_mail/inkit.md:44]:
```suggestion
![Request body code and webhook URL shown in the Braze webhook builder compose tab.][5]
```

File: _docs/_partners/message_orchestration/additional_channels/direct_mail/lob.md [hunk 10]
```diff
@@ -47,7 +50,7 @@ Lob requires an HTTP Header for authorization and an HTTP method. The following
   - **Authorization**: Basic `{{'<LOB_API_KEY>:' | base64_encode}}`
   - **Content-Type**: application/json
 
-![Lob preview][35]
+![An image of the request body code and webhook URL shown in the Braze webhook builder compose tab.][35]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/direct_mail/lob.md:53]:
```suggestion
![Request body code and webhook URL shown in the Braze webhook builder compose tab.][35]
```

File: _docs/_partners/message_orchestration/additional_channels/direct_mail/optilyz.md [hunk 11]
```diff
@@ -79,13 +79,13 @@ The `variation` field is optional and can define which design inside the automat
...
 
-![optilyz_compose][5]
+![An image of the request body code and webhook URL shown in the Braze webhook builder compose tab.][5]
 
 ### Step 2: Preview your request
 
 Next, preview your request in the left-hand panel or navigate to the __Test__ tab, where you can select a random user, an existing user, or customize your own to test your webhook. Remember to save your template before leaving the page!
 
-![optilyz_testing][7]
+![An image of the different testing fields available in the test tab of the Braze webhook builder.][7]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/direct_mail/optilyz.md:88]:
```suggestion
![Different testing fields available in the test tab of the Braze webhook builder.][7]
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/line.md [hunk 12]
```diff
@@ -60,40 +60,40 @@ In Braze, navigate to **Templates & Media > Content Blocks Library > + Create Co
...
+![An image the Content Block showing the Content Block name, Liquid tag, and censored channel access token.][2]
 
 Once you've set the channel access token inside a Content Block, you will be able to use the Line webhook template to send messages to users.
 
 ### Step 4: Select a webhook template
 
 From **Templates & Media**, go to **Webhook Templates** and choose one of the following Line Messenger webhook templates: 
 
-![Available templates]({% image_buster /assets/img_archive/line_templates.png %}){: style="border:0px;"}
+![An image of some of the available predesigned webhook templates in Braze.]({% image_buster /assets/img_archive/line_templates.png %}){: style="border:0px;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/messaging/line.md:71]:
```suggestion
![A selection of available predesigned webhook templates.]({% image_buster /assets/img_archive/line_templates.png %}){: style="border:0px;"}
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/line.md [hunk 13]
```diff
@@ -107,15 +107,15 @@ The custom attribute `Line ID` should be templated in the message body's `To:` f
...
+![An image of the test tab in the Braze webhook builder showing you can preview the message by sending it to an existing user.][3]
 
 If you receive the message successfully, you can start to configure its delivery settings.
 
 ## Using this integration
 
 Once set up, use this integration to target Line users. First, [create a segment][62] for all users for whom the `Line ID` exists as a custom attribute and turn on [analytics tracking][61] to track your Messenger subscription rates over time. 
 
-![Segment filter for Line IDs][63]
+![An image of the segment filter "line_id" set to "is not blank".][63]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/messaging/line.md:118]:
```suggestion
![Segment filter "line_id" set to "is not blank".][63]
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/messenger.md [hunk 14]
```diff
@@ -81,15 +81,15 @@ From **Templates & Media**, go to **Webhook Templates** and choose the **Faceboo
 
 Before you send your message, test your webhook. Make sure your Messenger ID is saved in Braze (or find it and test as a customized user), and use the preview to send the test message:
 
-![Sending a message to yourself][60]
+![An image of the test tab in the Facebook Messenger webhook template showing you can preview the message by sending it to an existing user.][60]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/messaging/messenger.md:84]:
```suggestion
![Test tab in the Facebook Messenger webhook template showing you can preview the message by sending it to an existing user.][60]
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/messenger.md [hunk 15]
```diff
@@ -81,15 +81,15 @@ From **Templates & Media**, go to **Webhook Templates** and choose the **Faceboo
...
 
 If you receive the message successfully, you can configure its delivery settings.
 
 ## Using this integration
 
 Once set up, use this integration to target Facebook Messenger users. If you are not sending messages using users' phone numbers and plan on sending Messenger messages repeatedly, you should [create a segment][62] for all users for whom the Messenger ID exists as a custom attribute and turn on [analytics tracking][61] to track your Messenger subscription rates over time. 
 
-![Segment filter for Messenger IDs][63]
+![An image of the segment filter "messenger_id" set to "is not blank".][63]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/messaging/messenger.md:92]:
```suggestion
![Segment filter "messenger_id" set to "is not blank".][63]
```

File: _docs/_partners/message_orchestration/additional_channels/retargeting/facebook.md [hunk 16]
```diff
@@ -39,19 +39,19 @@ If you are using this feature alongside the [Audience Sync to Facebook]({{site.b
 
 In the Braze dashboard, go to **Technology Partners** and select **Facebook**. In the Facebook Audience Export module, click **Connect Facebook**.
 
-![Activate Facebook][6]{: style="max-width:70%;"}
+![An image of the Facebook technology partners page in the Braze platform.][6]{: style="max-width:70%;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/retargeting/facebook.md:42]:
```suggestion
![Facebook technology partners page in the Braze platform.][6]{: style="max-width:70%;"}
```

File: _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md [hunk 17]
```diff
@@ -55,13 +55,13 @@ In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, yo
...
 
-![Partner connection][6]{: style="max-width:80%;"}
+![An image of the AppsFlyer audiences platform partner connection configuration page. The lower part of the images shows that the Braze external id box is checked.][6]{: style="max-width:80%;"}
 
 ### Step 4: Using AppsFlyer Audiences cohorts in Braze
 
 Once an AppsFlyer audience has been uploaded to Braze, you can use it as a filter when defining segments in Braze by selecting the __AppsFlyer Cohorts__ filter.
 
-![Cohort filter][7]
+![An image of the user attributes filter "AppsFlyer Cohorts" selected.][7]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md:64]:
```suggestion
![User attributes filter "AppsFlyer Cohorts" selected.][7]
```

File: _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md [hunk 18]
```diff
@@ -55,13 +55,13 @@ In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, yo
 3. Provide the data import key and Braze REST endpoint.
 4. Save the connection, and it will be available to link to any new or existing audience.
 
-![Partner connection][6]{: style="max-width:80%;"}
+![An image of the AppsFlyer audiences platform partner connection configuration page. The lower part of the images shows that the Braze external id box is checked.][6]{: style="max-width:80%;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md:58]:
```suggestion
![The AppsFlyer audiences platform partner connection configuration page. The lower part of the images shows that the Braze external id box is checked.][6]{: style="max-width:80%;"}
```

File: _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md [hunk 19]
```diff
@@ -46,7 +46,7 @@ NSDictionary *brazeInfo = @{
 
 ### Step 2: Get the Braze data import key
 
-In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![AppsFlyer Image][5]{: style="max-width:90%;"}
+In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![This image shows the "Data Import Using Cohort Import" box found in the AppsFlyer technology page. In this box, you are shown the data import key and the REST endpoint.][5]{: style="max-width:90%;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer_audiences.md:49]:
```suggestion
In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![The "Data Import Using Cohort Import" box on the AppsFlyer technology page. In this box, you are shown the data import key and the REST endpoint.][5]{: style="max-width:90%;"}
```

File: _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer.md [hunk 20]
```diff
@@ -90,7 +90,7 @@ Assuming you configure your integration as suggested above, Braze will map all o
 
 Your user base can be segmented by attribution data in the Braze dashboard using the Install Attribution filters.
 
-![User Attributes][2]
+![An image of four available filters. The first being "Install Atrribution Source is network_val_0". The second being "Install Atrribution Source is campaign_val_0". The third being "Install Atrribution Source is adgroup_val_0". The fourth being "Install Atrribution Source is creative_val_0". On the right side of the image, you can see how these attribution sources will be added to the user profile. In the "Install Attribution" box on a user's information page, Install Source is listed as network_val_0, campaign is listed as campaign_val_0, etc.][2]
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer.md:93]:
```suggestion
![Four available filters. The first is "Install Attribution Source is network_val_0". The second is "Install Attribution Source is campaign_val_0". The third is "Install Attribution Source is adgroup_val_0". The fourth is "Install Attribution Source is creative_val_0". On the right side of the image, you can see how these attribution sources will be added to the user profile. In the "Install Attribution" box on a user's information page, Install Source is listed as network_val_0, campaign is listed as campaign_val_0, etc.][2]
```

File: _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer.md [hunk 21]
```diff
@@ -57,7 +57,7 @@ AppsFlyer.setAdditionalData(customData);
 
 ### Step 2: Get the Braze data import key
 
-In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![AppsFlyer Image][4]{: style="max-width:70%;"}
+In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![An image showing the "Data Import for Install Atrribution" box available on the AppsFlyer Technology page. Inclued in this box is the data import key and the REST endpoint.][4]{: style="max-width:70%;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/attribution/appsflyer/appsflyer.md:60]:
```suggestion
In Braze, navigate to **Technology Partners** and select **AppsFlyer**. Here, you will find the REST Endpoint and generate your Braze data import key. Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in AppsFlyer's dashboard.<br><br>![The "Data Import for Install Attribution" box available on the AppsFlyer Technology page. Inclued in this box is the data import key and the REST endpoint.][4]{: style="max-width:70%;"}
```

File: _docs/_partners/message_orchestration/additional_channels/retargeting/facebook.md [hunk 22]
```diff
@@ -39,19 +39,19 @@ If you are using this feature alongside the [Audience Sync to Facebook]({{site.b
...
 
 A Facebook oAuth dialog window will appear to authorize Braze to create Custom Audiences into your Facebook ad accounts.
 
-![Facebook Dialog][8]{: style="max-width:30%;"}  ![Facebook Dialog][7]{: style="max-width:40%;"}
+![The first facebook dialogue box prompting to "Connect as X", where X is your Facebook username.][8]{: style="max-width:30%;"}  ![The second Facebook dialogue box prompting for permission to manage ads for your ad accounts.][7]{: style="max-width:40%;"}
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/retargeting/facebook.md:46]:
```suggestion
![The first facebook dialog box prompting to "Connect as X", where X is your Facebook username.][8]{: style="max-width:30%;"}  ![The second Facebook dialog box prompting for permission to manage ads for your ad accounts.][7]{: style="max-width:40%;"}
```

File: _docs/_partners/message_orchestration/additional_channels/mobile_wallet/passkit.md [hunk 23]
```diff
@@ -41,9 +40,9 @@ Once the data is passed into Braze, you can build audiences, personalize content
 
 To pass data from PassKit, please ensure that you have set your Braze external ID as PassKit’s `externalId`.
 
-1. Within **Settings**, under **Integrations** in your PassKit pass project or program click **Connect** under the **Braze** Tab.<br>![Settings Button][5]{: style="max-width:80%"}<br><br>
+1. Within **Settings**, under **Integrations** in your PassKit pass project or program click **Connect** under the **Braze** Tab.<br>![An image of the Braze integration tile in the PassKit platform.][5]{: style="max-width:80%"}<br><br>
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/additional_channels/mobile_wallet/passkit.md:43]:
```suggestion
1. Within **Settings**, under **Integrations** in your PassKit pass project or program click **Connect** under the **Braze** Tab.<br>![The Braze integration tile in the PassKit platform.][5]{: style="max-width:80%"}<br><br>
```

File: _docs/_partners/canvas_steps/google_audience_sync.md [hunk 24]
```diff
@@ -37,29 +37,29 @@ You will need to ensure that you have the following items created and/or complet
...
-![Google Sync]({% image_buster /assets/img/google_sync/google_sync5.png %})
+![The updated Google Ads technology page showing the "Add Mobile Advertising IDs" button checked, allowing you to same an iOS or Android AppID.]({% image_buster /assets/img/google_sync/google_sync5.png %})
 {% endalert %}
 
 ### Step 2: Add a Google Audience step in Canvas
 
 Add a step in your Canvas, select the dropdown at the top of the step, and select the __Google Audience__ step.
 
-![Google Sync][6]
+![An image of the steps to create a Google audience in Canvas.][6]
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/google_audience_sync.md:62]:
```suggestion
![Workflow of the previous steps to add a Google audience in Canvas.][6]
```

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 25]
```diff
@@ -68,7 +68,7 @@ For customers who have previously undergone the Facebook App Review process for
 
 Add a step in your Canvas, select the dropdown at the top of the step, and select __Facebook Audience Step__.
 
-![Add Audience Sync][11]{:style="max-width:70%"}
+![An image of what the steps written above look like in the Canvas UI.][11]{:style="max-width:70%"}
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:71]:
```suggestion
![Workflow of the previous steps to add a Facebook Audience step in Canvas.][11]{:style="max-width:70%"}
```

File: _docs/_partners/canvas_steps/facebook_audience_sync.md [hunk 26]
```diff
@@ -42,19 +42,19 @@ You will need to ensure that you have the following items created and/or complet
 
 In the Braze dashboard, go to __Technology Partners__ and select __Facebook__. In the Facebook Audience Export module, click __Connect Facebook__.
 
-![Activate Facebook][4]{: style="max-width:70%;"}
+![An image of the Facebook technology page in Braze.][4]{: style="max-width:70%;"}
```
Comment (lydia-xie) [inline _docs/_partners/canvas_steps/facebook_audience_sync.md:45]:
```suggestion
![Facebook technology page in Braze that includes an Overview module and Facebook Audience Export module with the Connected Facebook button.][4]{: style="max-width:70%;"}
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
The edits look great! :) I added some suggestions based on the feedback I got from Bre on my recent a11y PR (#3275). She mentioned to:
> Keep in mind that screen readers will prefix images with "Image of", and then read the alt text 

> Remember if the contents of the image are already covered somewhere else in the surrounding text, it's fine to use an empty alt tag! This tells screen readers to ignore the image

**Comment by KellieHawks** [comment]
@cla-bot check

**Review by lydia-xie — APPROVED** [review]
🎉

**Pull request merged**